src/types/login.ts

```diff
@@ -1,28 +1,23 @@
 import { Document, Model } from 'mongoose'
 
+import { FormBillingStatistic, LoginBase } from '../../shared/types/billing'
+
 import { IAgencySchema } from './agency'
-import { AuthType, IFormSchema, IPopulatedForm } from './form'
+import { IFormSchema, IPopulatedForm } from './form'
 import { IUserSchema } from './user'
 
-export interface ILogin {
+export interface ILogin extends LoginBase {
```

Comment:
Does `ILogin` still need to be exported? Looking at the rest of the codebase, it looks like the expectation is to only use `ILoginSchema`? (I only see `ILogin` being used in test, not in app code)

Reply:
We can probably stop exporting that; seems like the newer mongoose versions automatically type with a lean version of `xxxSchema` types when creating a new document. It used to be the case that we needed a separate typing for creating documents, as `xxxSchema` types contain extra props that is not needed to create a new document.